README.md

```diff
@@ -31,7 +31,10 @@ Include the CDN hosted script and configure the `<parts-kit>` custom element wit
 In the future, we'll have CMS plugins that automatically setup and configure of your parts kit.
 
 ```html
-<script type="module" src="https://unpkg.com/@viget/parts-kit@^0/lib/parts-kit.js"></script>
+<script
+  type="module"
+  src="https://unpkg.com/@viget/parts-kit@^0/lib/parts-kit.js"
+></script>
 <parts-kit config-url="YOUR-URL-HERE"></parts-kit>
```

Comment:
This is the part I was confused on. E.g. that it needs to be `/parts-kit/config.json` or `/parts-kit.json`. I tried lots of patterns to get the URL to where the file was located, but didn't realize I needed to include the `config.json` bit.

Reply:
For clarity, I'd suggest the following:

```suggestion
<parts-kit config-url="/path/to/config.json"></parts-kit>
```

Reply:
Yes, exactly!

Reply:
Good suggestion!